### PR TITLE
Update README.md with tryout instructions

### PR DESCRIPTION
**Pull request by datalogics-robl**
Add instructions on how to tryout the Java samples in IntelliJ

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,3 +30,10 @@ Licensing and Pricing options are customized to your usage and requirements. For
 - Printing
 - Security - Password, encrypt, watermark
 - Search for and replace or edit text
+
+## Instructions for trying out the samples
+- Clone this repository to your system (64-bit Intel Windows, 64-bit Intel Linux, or Apple Silicon)
+- Select a sample you would like to try
+- Open the folder containing the sample in [IntelliJ IDEA](https://www.jetbrains.com/idea/)
```
Comment (datalogics-kam) [inline README.md:37]: I think we need a step to remind users to press the Load Maven Project if this box appears.
image

File: README.md [hunk 2]
```diff
@@ -30,3 +30,10 @@ Licensing and Pricing options are customized to your usage and requirements. For
 - Printing
 - Security - Password, encrypt, watermark
 - Search for and replace or edit text
+
+## Instructions for trying out the samples
```
Comment (datalogics-kam) [inline README.md:34]: 👍

File: README.md [hunk 3]
```diff
@@ -30,3 +30,10 @@ Licensing and Pricing options are customized to your usage and requirements. For
...
+- Clone this repository to your system (64-bit Intel Windows, 64-bit Intel Linux, or Apple Silicon)
+- Select a sample you would like to try
+- Open the folder containing the sample in [IntelliJ IDEA](https://www.jetbrains.com/idea/)
+- From the Maven tab in the right sidebar, run the "Compile" goal
```
Comment (datalogics-kam) [inline README.md:38]: I couldn't find a way to make the IDE process the resources automatically (there's a button you can press in the Maven pane that will do that), but if we make changes like this to the run configurations, then all the user/evaluator has to do is to press the Run button. The IDE will run the `process-resources` phase automatically, and after it's run once, it's fast (the dependencies plugin can tell if the dependencies are already unpacked).

```diff
diff --git a/ContentCreation/AddElements/.idea/runConfigurations/AddElements.xml b/ContentCreation/AddElements/.idea/runConfigurations/AddElements.xml
index 265fb74..19777e8 100644
--- a/ContentCreation/AddElements/.idea/runConfigurations/AddElements.xml
+++ b/ContentCreation/AddElements/.idea/runConfigurations/AddElements.xml
@@ -7,6 +7,7 @@
     <module name="AddElements" />
     <option name="VM_PARAMETERS" value="-Djava.library.path=./target/lib" />
     <method v="2">
+      <option name="Maven.BeforeRunTask" enabled="true" file="$PROJECT_DIR$/pom.xml" goal="process-resources" />
       <option name="Make" enabled="true" />
     </method>
   </configuration>
```

(This is the button that generates the sources and puts the libraries in place)

image

File: README.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (datalogics-robl) [reply README.md]: Fixed

File: README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (datalogics-robl) [reply README.md]: Fixed

File: README.md (hunk 1 above)
Comment (datalogics-robl) [reply README.md:37]: This seems to be unique to your system, with only the one sample, I'll fix it when we have more info

File: README.md (hunk 3 above)
Comment (datalogics-robl) [reply README.md]: I like this idea, but I'll defer it to another pull request.

**Review by datalogics-josepha — APPROVED** [review]

**Pull request merged**